illumos-utils/src/smf_helper.rs

```diff
@@ -165,6 +165,38 @@ impl<'t> SmfHelper<'t> {
         Ok(())
     }
 
+    pub fn delpropvalue<P, V>(&self, prop: P, val: V) -> Result<(), Error>
```

Comment:
How does this differ from the existing `delpropvalue_default_instance` method below? There's only one (default) instance of this service in a zone AFAIU, so they seem like they should be equivalent. I could be missing something.